spec/factories/miq_user_role.rb

```diff
@@ -12,6 +12,16 @@
     name { |ur| ur.role ? "EvmRole-#{ur.role}" : generate(:miq_user_role_name) }
 
     after(:build) do |user, evaluator|
+      if evaluator.role.present?
+        system_roles = YAML.load_file(MiqUserRole::FIXTURE_YAML)
```

Comment:
Does this get loaded after each role we create?  Maybe MiqUserRole can memoize the system roles?